fetch.bs

```diff
@@ -7093,13 +7093,14 @@ descendants share a quota of 384 kibibytes. That value is computed as such:
 <p>To <dfn export>reserve deferred-fetch quota</dfn> for a <a>navigable container</a>
 <var>container</var> given an <a for=/>origin</a> <var>originToNavigateTo</var>:
 
-<p class=note>This is called when <var>container</var> and the document that initiated the
-navigation (the "source document") are not <a>same origin</a>. It potentially
-reserves either 64kb or 8kb of quota for the frame, if  the permissions policy allow. It is not
-observable to the cotnainer document whether the reserved quota was used in practice.
-This algorithm assumes that the container's document might delegate quota to the navigated frame,
-and the reserved quota would only apply in that case, and would be ignored if it ends up being
-shared.
+<p class=note>This is called on navigation, when the source document of the navigation is the
+<a for=/>navigable</a>'s parent document. It potentially reserves either 64kb or 8kb of quota for
```

Comment:
> It potentially reserves either 64kb or 8kb of quota for the frame

What is `frame` here? is it the `navigable` or `navigable container` or the `iframe`?

Reply:
For the container. Reworded